### DISCUSSION TITLE
How to access the release notes generated from a release pipeline

### DISCUSSION THREAD

**Discussion by progrock100** [rfennell/AzurePipelines#1540] [General]
Hi there, 

now I may be asking a stupid question here, please forgive me, I am new to devops.
I know I must be missing some basic knowledge regarding deployment in ADO.
### The problem
I am using the release notes generator task in my **release** pipeline and it appears to work fine, but I am not deploying anything anywhere.

Our software products are built internally and then we generally download the artifacts from the build pipeline and zip them up and email them or whatever.

I have the release notes task in the "deploy" stage in my release pipeline, its the last thing that gets run, and the logs show its working as expected. But, if I run the pipeline on the cloud, I can't access the release note that is generated. There is no way to do this via the ADO UI.
If I run on a local agent, I can manually log onto the local agent on our server and get it, so thats a workaround of some kind.

What does everyone else do when using these release pipelines ? It's probably something simple, but I can't work it out. I just want to be able to easily access the artifacts and the release note, download them and send to the customer for example.

Any advice would be appreciated. I am sorry for asking a non-release note generator question, just this seems like an active forum Mr Fennell is always engaging, so I thought I might get a quick and easy answer.

Thanks in advance.

### Background
I wanted to use release pipelines to handle releases, and release notes, as it's easy to just hit the "create release" button when we have a release ready (built and tested) in the master branch, and the release notes generator compares between the previous release, so it's exactly what we want.

(there is a way to do this without using a release pipeline, using tags I know, but that seems to be a bit of a pain, as I havent worked out yet how to automatically change the tag in to be used for compare in the yml, yeh I have a variable, but still that variable has to be set by someone which will be forgotten,  I can see it now).

**Comment by rfennell** [comment]
What people do with the generated note depends on their needs, some ship the markdown file, some convert the markdown to a PDF and other upload to a WIKI

Have a look at [this video](https://www.youtube.com/watch?v=xaV3dFoQdV8&t=332s) I did at a conference on how we use the task. That should help

**Reply by progrock100** [reply]
thanks I will take a look.
The conversion of the md file to a docx (maybe using pandoc) or a pdf is a secondary concern for now, we already have our own custom task that converts md to a branded company docx so we will probably use that in the release pipeline.
